src/renderer/components/app.scss

```diff
@@ -55,7 +55,7 @@
 }
 
 html {
-  font-size: 62.5%; // 1 rem == 10px
+  font-size: var(--font-size);
```

Comment:
Out of scope for this PR?

Reply:
I needed this to do the styling correctly for icons. This shouldn't break anything of ours since we don't use `rem`.

Reply:
Does not sound right, could you revert this and add more specific styling for icons?

Reply:
Sure, do you know why we want the rem to be smaller?